docs/includes/office-js-api-models.md

```diff
@@ -10,7 +10,10 @@ The Office JavaScript API includes two distinct models:
 
     This API model uses [promises](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise) and allows you to specify multiple operations in each request you send to the Office host. Batching operations in this manner can significantly improve add-in performance in Office on the web applications. Host-specific APIs were introduced with Office 2016 and cannot be used to interact with Office 2013.
```

Comment:
I think Promise needs a capital P? I could be wrong

Reply:
Capitalized only when talking about the class (singular), otherwise lower case. For example, the 2nd sentence of the official reference page says "To learn about the way promises work and how you can use them, we advise you to read _Using promises_ first.".